### ISSUE TITLE
pages/cli/vcluster_connect out of date with changes from cmd/vclusterctl/cmd/connect.go (2024-02-16T06:42:31.000Z)

### ISSUE THREAD

**Issue by avb-is-me** [avb-is-me/vcluster#10]
Based on reviewing the updated connect command code and the existing documentation, I would recommend updating the documentation to reflect some of the new flags and functionality added in the code.

Here are some updates I would suggest:

1. Document the new `--extensions` flag which allows custom extensions to the vcluster.

2. Document the new `--background-proxy` flag which creates a background proxy in Docker.

3. Document the new `--project` flag for associating vclusters with a Pro project.

4. Update the flags section to include the new flags added in the code - `--extensions`, `--background-proxy`, `--project`, etc.

Here is an example updated flags section snippet showing some of the new flags:

```
--background-proxy                  If specified, vcluster will create the background proxy in docker [its mainly used for vclusters with no nodeport service.]
--extensions                        When enabled allow custom extensions to vcluster, and it is super cool  
--project string                    [PRO] The pro project the vcluster is in
```

In summary, while the core functionality remains the same, there are several useful new flags and capabilities added that should be documented to help users.

My recommendation would be:

"DEV_DOCS_REQUIRE_UPDATE"
